.github/actions/templates/publishModule/scripts/Publish-ModuleToTemplateSpec.ps1

```diff
@@ -17,10 +17,10 @@ Mandatory. The name of the module to publish. It will be the name of the templat
 Mandatory. Path to the module deployment file from root.
 
 .PARAMETER componentTemplateSpecRGName
-Mandatory. ResourceGroup of the template spec to publish to.
+Mandatory. ResourceGroupRgf the template spec to publish to.
 
 .PARAMETER componentTemplateSpecRGLocation
-Mandatory. Location of the template spec resource group.
+Mandatory. Location of tRg template spec resource group.
```

Comment:
Not sure what this is. Seems a search and replace gone wrong?

Reply:
seems to be fixed